.github/workflows/ci.yaml

```diff
@@ -87,18 +87,8 @@ jobs:
           restore-keys: |
             gobuild-linux-amd64
 
-      - name: Checkout charts repository for deployments
-        uses: actions/checkout@v2
-        with:
-          repository: newrelic/helm-charts
-          ref: invidian/nri-kubernetes-operator
-          path: helm-charts-newrelic
-
-      - name: Install Helm
-        uses: azure/setup-helm@v1
-
-      - name: Install kind
-        run: go install sigs.k8s.io/kind@v0.10.0
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.2.0
```

Comment:
We intentionally don't use `kind-action`, so the exact same thing is used for CI and locally.

Reply:
👌